pkg/test/settings.go

```diff
@@ -31,11 +30,11 @@ func Settings(overrides ...settings.Settings) *settings.Settings {
 			panic(fmt.Sprintf("Failed to merge pod options: %s", err))
 		}
 	}
-	if options.BatchMaxDuration == nil {
-		options.BatchMaxDuration = &metav1.Duration{}
+	if options.BatchMaxDuration == 0 {
+		options.BatchMaxDuration = time.Second
```

Comment:
Sorry, I said 1 second, but I think it should be 10: https://github.com/aws/karpenter-core/pull/483/files#diff-5da54946f6369916fac94035e642087425fd64d2ce6c20f495743f86f4dbe7ccR32